src/Common/Storage/Commands.Storage/Table/Cmdlet/NewAzureStorageTableSasToken.cs

```diff
@@ -55,6 +56,12 @@ public string Policy
             ParameterSetName = SasPermissionParameterSet)]
         public string Permission { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Protocol can be used in the request with this SAS token.")]
+        public SharedAccessProtocol Protocol { get; set; }
```

Comment:
SharedAccessAccountServices , SharedAccessAccountResourceTypes , SharedAccessProtocol is an Enum, and Powershell support Enum well. Use can just tab to get the enum value, don't need to create by themselves.